package.json

```diff
@@ -2437,10 +2448,9 @@
         }
     },
     "engines": {
-        "vscode": "^1.41.0"
+        "vscode": "^1.42.0"
```

Comment:
Is this for `ThemeIcon` or is there something else?

Reply:
Also, if possible I'd suggest splitting the `vscode` vs `vscode-test` stuff out into another PR; if this depends on that, then we can do that one first and the merge into here.

Reply:
Reverted this change.